discord/ext/bridge/bot.py

```diff
@@ -106,6 +117,54 @@ def decorator(func) -> BridgeCommandGroup:
 
         return decorator
 
+    async def invoke(self, ctx: ExtContext | BridgeExtContext):
+        if ctx.command is not None:
+            self.dispatch("command", ctx)
+            if br_cmd := isinstance(ctx.command, BridgeExtCommand):
+                self.dispatch("bridge_command", ctx)
```

Comment:
Should the text-based commands be called as well for consistency with application commands?